src/clj/witan/send/model/output.clj

```diff
@@ -85,8 +85,24 @@
                                    :beta (- (get-in population [calendar-year academic-year]) j)})))
                     coll years)) {} ages))
 
-(defn output-transitions [file projections]
-  (binding [*print-length* nil] (spit file (pr-str projections))))
+(defn output-transitions
+  ([dir filename projections]
+   (binding [*print-length* nil] (spit (str dir "/" filename ".edn") (pr-str projections))))
+  ([dir filename projections simulations]
+   (with-open [writer (io/writer (io/file (str dir "/" filename ".csv")))]
+     (let [columns [:calendar-year :setting-1 :need-1 :academic-year-1 :setting-2 :need-2 :academic-year-2 :avg-number-of]
+           headers (mapv name columns)
+           parse-need-setting #(map name ((comp reverse states/split-need-setting) %))
+           rows (sort (remove empty?
+                              (for [[[cy ay need-setting-source need-setting-destination] transitions] projections]
+                                (if (> (m/round0 (/ transitions simulations)) 0)
+                                  (into [] (flatten [cy
+                                                     ay
+                                                     (parse-need-setting need-setting-source)
+                                                     (inc ay)
+                                                     (parse-need-setting need-setting-destination)
+                                                     (m/round0 (/ transitions simulations))]))))))]
+       (csv/write-csv writer (into [headers] rows))))))
```

Comment:
Do we need the original one of these still? the .edn version?

Reply:
Can't hurt to have it.  Will make a handy comparison if we wanted to test.  Neither are completely accurate at the moment but those fixes are to come.

Reply:
Not accurate, in that they don't capture everything?

Reply:
And too much!  And oddly.

Reply:
Should all be in the blogs/other big PR.

Reply:
okay. I'm not sure we need both, but it doesn't really hurt. I don't like however that the same fn does two very different things if you use 3 or 4 args. Could you separate them and call them different things please?